FluentTerminal.App/Strings/zh-CN/Resources.resw

```diff
@@ -785,7 +785,7 @@ Fuzzy</comment>
     <comment>Command
 Fuzzy</comment>
   </data>
-  <data name="NewSshTab.Text" xml:space="preserve">
+  <data name="NewRemoteTab.Text" xml:space="preserve">
```

Comment:
this has been renamed to NewSshTab with #410

Reply:
The same answer as above...

Reply:
Fixed.